pkg/apis/serving/v1beta1/explainer_alibi.go

```diff
@@ -42,19 +42,22 @@ const (
 
 // AlibiExplainerSpec defines the arguments for configuring an Alibi Explanation Server
 type AlibiExplainerSpec struct {
-	// The type of Alibi explainer
-	// Valid values are:
-	// - "AnchorTabular";
-	// - "AnchorImages";
-	// - "AnchorText";
-	// - "Counterfactuals";
-	// - "Contrastive";
+	// The type of Alibi explainer <br />
```

Comment:
So new lines are not added to generated docs? This seems annoying we need to do everywhere.

Reply:
it is a bit annoying but most of fields have single line comment.